docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+# USB Camera Device Service RTSP Streaming Guide
+
+## Contents
+
+[System Requirements](#system-requirements)  
+[How It Works](#how-it-works)  
```

Comment:
Do we need this? Should README.md be merged in here to give the overview / architecture?